gateway/src/http_entrypoints.rs

```diff
@@ -1378,6 +1378,25 @@ async fn recovery_host_phase2_upload(
     Ok(HttpResponseOk(HostPhase2RecoveryImageId { sha256_hash }))
 }
 
+/// Get the identifier for the switch this MGS instance is connected to.
+///
+/// Note that most MGS endpoints behave identically regardless of which scrimlet
+/// the MGS instance is running on; this one, however, is intentionally
+/// different.
+#[endpoint {
+    method = GET,
+    path = "/local/switch-id",
+}]
+async fn sp_local(
```

Comment:
Maybe include `switch` in the method name? I don't know MGS too well, but it also reports data about the local Gimlet's SP right? If that's true, including `switch` would help disambiguate.

Reply:
Yes, good call. Renamed in d4267b38601022b8d0c2ae660c239e045dea7a47